docs/developer-guide/advanced-project-customization.md

```diff
@@ -0,0 +1,220 @@
+                <deployables>
+                    <deployable>
+                        <groupId>${project.groupId}</groupId>
+                        <artifactId>${project.artifactId}</artifactId>
+                        <type>war</type>
+                        <properties>
+                            <context>/mapstore</context>
+                        </properties>
+                    </deployable>
+                </deployables>
+            </configuration>
+        </plugin>
+```
+
+In order to setup the multiple installations we can, for example, create three folders inside our `datadir`, denoting three different installations.
+
+```text
+├── datadir
+    ├── customA
+    ├── customB
+    ├── customC
+```
+
+Please note that the only requirement regarding folder structure is that the `datadir` exists, any content inside is optional and can be structured freely.
+
+As a final step, we will re-set the `datadir.location` system property outlined above to point to a specific installation, such that:
+
+> java -Ddatadir.location=/etc/mapstore/datadir/customA
+
+## Customize confiuration, assets and translations
+
+Inside our example installation folder `datadir/customA` we can include customizations pertinent to this instance only and excluded from the other two instances exemplified in the section above.
+
+For the purpose of this guideline we will adhere to the following folder structure convention, as outlined [here](https://docs.mapstore.geosolutionsgroup.com/en/latest/developer-guide/externalized-configuration/#using-a-data-directory):
+
+```text
+├── customA
+    ├── configs
+    │   └── localConfig.json.patch
+    └── extensions
+        └── customizations
+            ├── assets
+            │   ├── css
+            │   │   └── custom-style.css
+            │   ├── img
+            │   │   └── custom-logo.jpg
+            │   │              
+            │   └── js
+                    └── customPlugin.js
+            ├── index.js
+            ├── index.js.LICENSE.txt
+   
+            ├── index.json
+            └── translations
+                ├── data.en-US.json
+            │   ├── ...other translation files.
+```
+
+In the `customA/configs/localConfig.json.patch` file, we can patch the base configuration that is supplied by the `configs/localConfig.json` file inside root of the custom project, effectively customizing for this particular installation.
+
+To include both these files into your build, import the `setLocalConfigurationFile` helper function provided by MapStore inside `js/app.jsx` and call it by providing the paths to both files:
+
+```js
+import { setLocalConfigurationFile} from '@mapstore/utils/ConfigUtils';
+...
+
+setLocalConfigurationFile(['configs/localConfig.json', 'configs/localConfig.patch.json']);
+```
+
+The `localConfig.json.patch` file can serve as the central hub for these customizations (see example below), essentially linking the whole content of the custom datadir folder and allowing for:
```

Comment:
```suggestion
This alternative version  uses [`@mapstore/patcher`](https://github.com/geosolutions-it/Patcher) and allow perform more complex overrides, even if requires this change to be applied to the code, so it will not work with the standard product. 

The `localConfig.json.patch` file can serve as the central hub for these customizations (see example below), essentially linking the whole content of the custom datadir folder and allowing for:
```